### ISSUE TITLE
bower install does not work

### ISSUE THREAD

**Issue by xplicit** [steveathon/bootstrap-wysiwyg#28]
When I try to install using bower into empty dir, I've got an error and installation fails.

    bower install bootstrap-wysiwyg-steveathon

    bower cached        git://github.com/steveathon/bootstrap-wysiwyg.git#1.0.2
    bower validate      1.0.2 against git://github.com/steveathon/bootstrap-wysiwyg.git#*
    bower not-cached    git://github.com/tzuryby/jquery.hotkeys.git#>= 1.0
    bower resolve       git://github.com/tzuryby/jquery.hotkeys.git#>= 1.0
    bower cached        git://github.com/FortAwesome/Font-Awesome.git#4.3.0
    bower validate      4.3.0 against git://github.com/FortAwesome/Font-Awesome.git#>= 3.0.0
    bower cached        git://github.com/tcollard/google-code-prettify.git#1.0.4
    bower validate      1.0.4 against git://github.com/tcollard/google-code-prettify.git#>= 1.0.1
    bower cached        git://github.com/jquery/jquery.git#2.1.3
    bower validate      2.1.3 against git://github.com/jquery/jquery.git#>= 1.9.1
    bower cached        git://github.com/twbs/bootstrap.git#3.3.2
    bower validate      3.3.2 against git://github.com/twbs/bootstrap.git#>= 2.0
    bower new           version for git://github.com/jquery/jquery.git#>= 1.9.1
    bower resolve       git://github.com/jquery/jquery.git#>= 1.9.1
    bower download      https://github.com/jquery/jquery/archive/2.1.4.tar.gz
    bower new           version for git://github.com/twbs/bootstrap.git#>= 2.0
    bower resolve       git://github.com/twbs/bootstrap.git#>= 2.0
    bower download      https://github.com/twbs/bootstrap/archive/v3.3.4.tar.gz
    bower extract       jquery#>= 1.9.1 archive.tar.gz
    bower resolved      git://github.com/jquery/jquery.git#2.1.4
    bower extract       bootstrap#>= 2.0 archive.tar.gz
    bower resolved      git://github.com/twbs/bootstrap.git#3.3.4
    bower ENORESTARGET  No tag found that was able to satisfy >= 1.0
    
    Additional error details: 
    No versions found in git://github.com/tzuryby/jquery.hotkeys.git

**Comment by steveathon** [comment]
Tagging a release after the next PR merge.

**Comment by RandomlyKnighted** [comment]
@steveathon I can confirm that this issue still exists even after tagging the newest release. Any ideas as to how we can fix this?

**Comment by RandomlyKnighted** [comment]
@xplicit I've just tested the latest release and this issue has been fixed.

**Comment by rohan-deshpande** [comment]
I'm trying to install via bower and keep getting the mindmup version.

**Comment by RandomlyKnighted** [comment]
what command are you using?

**Comment by rohan-deshpande** [comment]
Hmm I was using `bower install bootstrap-wysiwyg` but I uninstalled, cleared the cache and reinstalled and then it was fine.

**Comment by RandomlyKnighted** [comment]
The package for this project is actually named `bootstrap-wysiwyg-steveathon`

**Comment by rohan-deshpande** [comment]
Hmm but that's not what it says in your name prop in the bower.json file on the master branch. Might be a good idea to update that.

**Comment by RandomlyKnighted** [comment]
Yeah it's an inconsistency that shouldn't be there. The package name in the bower.json file is correct, it's just Bower has it wrong in their directory. Until it's fixed `bower install bootstrap-wysiwyg-steveathon` should resolve the issue. I've already asked about how I can go about getting it fixed. I'm just waiting on a response from someone on the Bower team.

**Comment by rohan-deshpande** [comment]
Interesting, so I just made a new bower.json and put `"bootstrap-wysiwyg-steveathon":"~1.0.4"` as a dependency and it worked fine. So I guess bower thinks that is the package name, which sort of makes sense because the lib you forked this from already seems to have `bootstrap-wysiwyg` registered to their repo.

**Comment by RandomlyKnighted** [comment]
Actually as you can see [here](https://github.com/mindmup/bootstrap-wysiwyg) Mindmup never used Bower in their repo. The [original commit](https://github.com/steveathon/bootstrap-wysiwyg/commit/c7b3b1bf74747923812d23e8147162aeaf4d51b6) for adding Bower wasn't added until April 1, 2014 and Mindmup's last commit (other than updating the README) was on July 29, 2013. So really this repo registered the name first. I'm still not sure why Bower has the wrong package name for it but we're working on it.

**Comment by rohan-deshpande** [comment]
Ah right! Very odd that bower could somehow download the wrong repo somehow...
